test/distributed/fsdp/test_fsdp_optim_state.py

```diff
@@ -628,8 +700,9 @@ def test_shard_full_optim_state_dict_unmanaged_params(
           should be no error (emulating model parallel use cases where some
           parameters may be managed externally to FSDP).
         We do not separately test unmanaged parameters for
-        :meth:`scatter_full_optim_state_dict` to save CI cost since it calls
-        into the same subroutine :meth:`_flatten_full_optim_state_dict`.
+        :meth:`scatter_full_optim_state_dict` and `flatten_sharded_optim_state_dict`
+        to save CI cost since it call into the same subroutine
+        :meth:`_flatten_full_optim_state_dict`.
```

Comment:
```suggestion
        :meth:`_flatten_optim_state_dict`.
```

(since you renamed it)